client/components/Main.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import FeaturedProd from "./FeaturedProd";
+import "./styles/main.css";
+import familyPhoto from "../../public/photos/BellFamilyPhoto.jpeg";
```

Comment:
Since these images are being exposed as static files from the server in the public folder, you might be better off putting their source directly in the image tag's `src` attribute, rather than bundling the image here. Since you ended up adding the image loader to webpack this works fine, so ultimately not a problem, just a suggestion